.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: go
 go:
-  - "1.11"
+  - "1.13"
```

Comment:
From my understanding of https://github.com/travis-ci/gimme#asking-gimme-what-a-version-is, specifying `1.13.x` would automatically update to newer patch versions, would it make sense to use that?

Reply:
Good idea!